services/QuillLMS/app/assets/stylesheets/pages/request_quote.scss

```diff
@@ -0,0 +1,7 @@
+.quote-request-form {
+  width: 600px;
```

Comment:
It probably doesn't matter much, but I think we should be in the habit of writing responsive CSS, especially on simple pages like this that shouldn't require an alternate design. One easy way to do that is usually just to do `max-width: 600px; width: 100%;` inside of specifying an exact width value.